dev-python/exdown/metadata.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="project">
+		<email>sci@gentoo.org</email>
+		<name>Gentoo Science Project</name>
+	</maintainer>
```

Comment:
Sorry for not noticing this at first, but we can't do this, unless it's been acked by a member of sci project. Chances are they'd rather dump tikzplotlib to m-n than take on more maintenance burden.

Reply:
I'll just add myself as maintainer than